src/BinSkim.Rules/PERules/BA2014.DoNotDisableStackProtectionForFunctions.cs

```diff
@@ -59,6 +59,9 @@ private static StringSet BuildApprovedFunctionsStringSet()
                 "_TlgWrite",
                 "__vcrt_trace_logging_provider::_TlgWrite"
             };
+
+            // Also allow Gs initialization functions.
+            result.UnionWith(StackProtectionUtilities.GSInitializationFunctionNames);
```

Comment:
>StackProtectionUtilities.GSInitializationFunctionNames [](http://example.com/codeflow?start=29&length=54)

tried can not put this in to StringSet(){}

Reply:
Oops, of course! Because it is a collection, sorry for the dumb advice. :)